google-cloud-spanner/src/test/java/com/google/cloud/spanner/DatabaseClientImplTest.java

```diff
@@ -1053,10 +1053,10 @@ public void testPartitionedDmlDoesNotTimeout() {
     mockSpanner.setExecuteSqlExecutionTime(SimulatedExecutionTime.ofMinimumAndRandomTime(10, 0));
```

Comment:
I think you need to increase this value instead of the values below. This is the time that the server will wait to simulate that the DML statement takes a 'long' time. The values below are the times that the client should wait before throwing a deadline exceeded error.

Reply:
Thanks for clarifying.